core/templates/pages/contributor-dashboard-page/modal-templates/question-suggestion-review.component.html

```
@@ -70,11 +70,11 @@ <h3 class="oppia-mobile-question-header" title="{{questionHeader}}">
 
 <div class="modal-footer">
   <div class="oppia-review-buttons">
-    <div ngbTooltip="Enter review message before rejecting" placement="bottom" disableTooltip="!(reviewMessage.length == 0)">
+    <div ngbTooltip="Enter review message before rejecting" placement="bottom" [disableTooltip]="!(reviewMessage.length == 0)">
       <button class="btn btn-danger e2e-test-question-suggestion-review-reject-button"
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)